src/core/core.helpers.js

```diff
@@ -669,6 +669,11 @@ module.exports = function(Chart) {
 	};
 	// Request animation polyfill - http://www.paulirish.com/2011/requestanimationframe-for-smart-animating/
 	helpers.requestAnimFrame = (function() {
+		if (typeof window === 'undefined') {
```

Comment:
> What about the other specific `HTMLCanvasElement`, `CanvasRenderingContext2D`, `CanvasGradient` or even the `iframe`?

Since we don't encounter any other problems, I believe those parts are not triggered on script evaluation, and that is fine with our server side.